openedx/core/djangoapps/user_api/tests/test_helpers.py

```diff
@@ -214,3 +215,47 @@ def stub_view(request):  # pylint: disable=missing-docstring
             self.captured_request = request
             return response
         return shim_student_view(stub_view, check_logged_in=check_logged_in)
+
+
+MOVIE_MIN_LEN = 3
+MOVIE_MAX_LEN = 100
```

Comment:
Nit: I'd prefer to see these moved inside the `TestCaseForm` class. You could also read them out as `TestCaseForm.favorite_move.min_length` and not bother defining them as constants at all - they don't seem that important.